src/host/os_getversion.c

```diff
@@ -55,7 +55,9 @@ int os_getversion(lua_State* L)
 
 #if defined(PLATFORM_WINDOWS)
 
+#ifdef _MSC_VER
```

Comment:
Was this causing build issues? If not, I'd consider it out of scope of this PR.

Reply:
Compiler warning:

```
../../src/host/os_getversion.c:58: warning: ignoring '#pragma comment ' [-Wunknown-pragmas]
   58 | #pragma comment(lib, "version.lib")
      |
```